addons/packages/pinniped/0.12.1/bundle/config/values.star

```diff
@@ -150,6 +156,23 @@ def validate_static_client() :
   end
 end
 
+def is_mgmt_cluster():
```

Comment:
I opted to go for funcs instead of inline value checks to hopefully keep our templates easier to reason about.